test/e2e/authorization.go

```diff
@@ -0,0 +1,572 @@
+			})
+		})
+
+		g.When("the default service account is in non-default namespace", func() {
+			g.BeforeEach(func() {
+				tc.data.users = []string{"system:serviceaccount:non-default:default"}
+			})
+			g.It("should deny to list StatefulSets", func() {
+				tc.data.resources = []string{"apps/statefulsets"}
+				tc.data.verbs = []string{"list"}
+				tc.run(context.TODO(), cs, false)
+				gomega.Expect(tc.output.passed).To(gomega.BeTrue(), tc.output.String())
+			})
+		})
+
+		g.When("the service account is persistent-volume-binder", func() {
+			g.BeforeEach(func() {
+				tc.data.users = []string{"system:serviceaccount:kube-system:persistent-volume-binder"}
+				tc.data.groups = [][]string{{"system:serviceaccounts:kube-system"}}
+				tc.data.namespaces = []string{"kube-system"}
+			})
+			g.It("should allow to update PersistentVolumeClaims", func() {
+				tc.data.resources = []string{"persistentvolumeclaims"}
+				tc.data.verbs = []string{"update"}
+				tc.run(context.TODO(), cs, true)
+				gomega.Expect(tc.output.passed).To(gomega.BeTrue(), tc.output.String())
+			})
+			g.It("should allow to create PersistentVolumes", func() {
+				tc.data.resources = []string{"persistentvolumes"}
+				tc.data.verbs = []string{"create"}
+				tc.run(context.TODO(), cs, true)
+				gomega.Expect(tc.output.passed).To(gomega.BeTrue(), tc.output.String())
+			})
+
+		})
+
+		g.When("the service account is aws-cloud-provider", func() {
+			g.BeforeEach(func() {
+				tc.data.users = []string{"system:serviceaccount:kube-system:aws-cloud-provider"}
+				tc.data.groups = [][]string{{"system:serviceaccounts:kube-system"}}
+			})
+			g.It("should allow to patch Nodes", func() {
+				tc.data.resources = []string{"nodes"}
+				tc.data.verbs = []string{"patch"}
+				tc.run(context.TODO(), cs, true)
+				gomega.Expect(tc.output.passed).To(gomega.BeTrue(), tc.output.String())
+			})
+		})
+
+		g.When("the service account is api-monitoring-controller", func() {
+			g.BeforeEach(func() {
+				tc.data.users = []string{"system:serviceaccount:api-infrastructure:api-monitoring-controller"}
+			})
+			g.When("the namespace is kube-system", func() {
+				g.BeforeEach(func() {
+					tc.data.namespaces = []string{"kube-system"}
+				})
+				g.It("should allow to update 'skipper-default-filters' ConfigMap", func() {
+					tc.data.resources = []string{"configmaps"}
+					tc.data.verbs = []string{"update"}
+					tc.data.names = []string{"skipper-default-filters"}
+					tc.run(context.TODO(), cs, true)
+					gomega.Expect(tc.output.passed).To(gomega.BeTrue(), tc.output.String())
+				})
+				g.It("should deny to update any other ConfigMap", func() {
+					tc.data.resources = []string{"configmaps"}
+					tc.data.verbs = []string{"update"}
+					// Technically, this should result in access undecided because we allow
+					// access to 'skipper-default-filters' ConfigMap only and we haven't
+					// specified a resource name in the test case.
```

Comment:
Can we just specify another resource name?

Reply:
I can do that.

Reply:
I actually tried specifying `foo-configmap` but it's still undecided. Not sure why this is happening. Should I still add the name?